plugin/mlsdbprovider.cpp

```diff
@@ -305,7 +317,6 @@ void MlsdbProvider::calculatePositionAndEmitLocation()
 void MlsdbProvider::tryFetchOnlinePosition()
```

Comment:
Why this is separate function, could be same as the calculatePositionAndEmitLocation?

Reply:
Indeed should be, I'll fix this in the cleanup commit